daphne/src/messages/mod.rs

```diff
@@ -412,14 +412,53 @@ impl TryFrom<Query> for BatchSelector {
     }
 }
 
+/// The PrepareInit message consisting of the report share and the Leader's initial prep share.
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct PrepareInit {
+    pub report_share: ReportShare,
+    pub draft05_payload: Option<Vec<u8>>,
```

Comment:
draft_05?

Reply:
My question is more "why draft 5" and not "7", this is now the only mention of draft 5 in the code

Reply:
😮‍💨 Yeah that was just my dumb brain. Should be 7 -- fixed.